appinventor/appengine/src/com/google/appinventor/client/youngandroid/YoungAndroidFormUpgrader.java

```diff
@@ -1563,12 +1562,13 @@ private static void upgradeWarnDialog(String aMessage) {
     message.setStyleName("DialogBox-message");
     VerticalPanel vPanel = new VerticalPanel();
     Button okButton = new Button("OK");
-    okButton.addClickListener(new ClickListener() {
-        @Override
-        public void onClick(Widget sender) {
-          dialogBox.hide();
-        }
-      });
+    okButton.addClickHandler(new ClickHandler(){
+    
```

Comment:
I suggest removing the blank line here to keep this similar to the previous version.